sdk/recoveryservicesdatareplication/arm-recoveryservicesdatareplication/review/arm-recoveryservicesdatareplication.api.md

```diff
@@ -111,202 +89,88 @@ export interface CheckNameAvailabilityResponseModel {
-    get(resourceGroupName: string, vaultName: string, emailConfigurationName: string, options?: EmailConfigurationGetOptionalParams): Promise<EmailConfigurationGetResponse>;
-    list(resourceGroupName: string, vaultName: string, options?: EmailConfigurationListOptionalParams): PagedAsyncIterableIterator<EmailConfigurationModel>;
+export interface DeploymentPreflightPostOptionalParams extends OperationOptions {
+    body?: DeploymentPreflightModel;
 }
 
 // @public
-export interface EmailConfigurationCreateOptionalParams extends coreClient.OperationOptions {
-    body?: EmailConfigurationModel;
+export interface DeploymentPreflightResource {
+    apiVersion?: string;
+    location?: string;
+    name?: string;
+    properties?: any;
+    type?: string;
 }
 
 // @public
-export type EmailConfigurationCreateResponse = EmailConfigurationModel;
-
-// @public
-export interface EmailConfigurationGetOptionalParams extends coreClient.OperationOptions {
+export interface DiskControllerInputs {
+    controllerId: number;
+    controllerLocation: number;
+    controllerName: string;
 }
 
 // @public
-export type EmailConfigurationGetResponse = EmailConfigurationModel;
-
-// @public
-export interface EmailConfigurationListNextOptionalParams extends coreClient.OperationOptions {
+export interface EmailConfigurationCreateOptionalParams extends OperationOptions {
 }
 
 // @public
-export type EmailConfigurationListNextResponse = EmailConfigurationModelCollection;
-
-// @public
-export interface EmailConfigurationListOptionalParams extends coreClient.OperationOptions {
+export interface EmailConfigurationGetOptionalParams extends OperationOptions {
 }
 
 // @public
-export type EmailConfigurationListResponse = EmailConfigurationModelCollection;
-
-// @public
-export interface EmailConfigurationModel {
-    readonly id?: string;
-    readonly name?: string;
-    properties: EmailConfigurationModelProperties;
-    readonly systemData?: EmailConfigurationModelSystemData;
-    readonly type?: string;
+export interface EmailConfigurationListOptionalParams extends OperationOptions {
 }
 
 // @public
-export interface EmailConfigurationModelCollection {
-    nextLink?: string;
-    value?: EmailConfigurationModel[];
+export interface EmailConfigurationModel extends ProxyResource {
+    properties?: EmailConfigurationModelProperties;
 }
 
 // @public
 export interface EmailConfigurationModelProperties {
     customEmailAddresses?: string[];
     locale?: string;
+    readonly provisioningState?: ProvisioningState;
     sendToOwners: boolean;
 }
 
-// @public (undocumented)
-export interface EmailConfigurationModelSystemData extends SystemDataModel {
+// @public
+export interface EmailConfigurationOperations {
+    create: (resourceGroupName: string, vaultName: string, emailConfigurationName: string, resource: EmailConfigurationModel, options?: EmailConfigurationCreateOptionalParams) => Promise<EmailConfigurationModel>;
```

Comment:
why this is not LRO?

Reply:
it uses sync not async in typespec:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/recoveryservicesdatareplication/DataReplication.Management/EmailConfigurationModel.tsp#L45

Reply:
could you double confirm for old swagger is not a LRO?

Reply:
yes, in their old swagger, it's non-lro
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/recoveryservicesdatareplication/resource-manager/Microsoft.DataReplication/preview/2021-02-16-preview/recoveryservicesdatareplication.json#L339-L403